### ISSUE TITLE
Exercise GO API devops documentation

### ISSUE THREAD

**Issue by kltm** [geneontology/go-fastapi#70]
The deployment documentation needs to be exercised such that a GO developer familiar with the overall project and who has access to the credential store should be able to

1. deploy a new verion of the API into AWS (for testing, etc)
2. switch between a currently deployed production version and a new version

**Comment by kltm** [comment]
This is currently cycling between @sierra-moxon and @abessiari . Once they are complete, we will pass this on to @dustine32 for a final look.

**Comment by kltm** [comment]
This is the documentation and "debugging" cycle for #14 .

**Comment by kltm** [comment]
@dustine32 , @sierra-moxon has successfully put out a couple versions of the GO API using the docs. As a final step, would it be possible for you to give it a try next week? I'd be on hand to answer any questions. If not possible, no problem--we'll revisit later on.

**Comment by kltm** [comment]
@dustine32 and myself gave the documentation a try and, while we were able to get some parts working, had trouble getting the effects we wanted. In general, we're thinking that it might be good to refactor the doc into three parts:

1. devops environment setup; may be generally useful for other projects as well
2. general utility and terraform use; may be useful for other projects as well
3. GO API-specific test and deployment workflow, including names, how to get a fix out, etc.; includes how we want to run devops, names to use, etc.

We've added this to the agenda for the next general go software call.

Will follow up with https://github.com/geneontology/go-fastapi/issues/78
